automation_tools/__init__.py

```diff
@@ -1003,21 +1003,19 @@ def cdn_install(run_katello_installer=True):
 
     The following environment variables affect this command:
 
-    RHN_USERNAME
-        Red Hat Network username.
-    RHN_PASSWORD
-        Red Hat Network password.
-    RHN_POOLID
-        Optional. Red Hat Network pool ID. Determines what software will be
-        available from RHN.
     ADMIN_PASSWORD
         Optional, defaults to 'changeme'. Foreman admin password.
+    SATELLITE_VERSION
+        Satellite version.
 
     """
     admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
 
     # Install required packages for the installation
-    run('yum install -y katello')
+    if os.environ.get('SATELLITE_VERSION') == '6.2':
```

Comment:
Since SATELLITE_VERSION is a required env var the task should fail it is not defined. Otherwise it will try to run katello whenever it is not defined and the user maybe be willing to install 6.2